app/views/carts/show.html.erb

```diff
@@ -43,7 +43,7 @@
         Total Price: <%= number_to_currency(@cart.cart_total_price) %>
     </h1>
     <% if current_user %>
-      <%= button_to "Checkout", orders_path(current_user), method: :post, class: 'checkout'%>
+      <%= button_to "Checkout", orders_path(current_user, cart: @cart.contents), method: :post, class: 'checkout'%>
```

Comment:
We no longer need cart as a custom param... lets torch it

Reply:
Okay, it's been torched!